crates/bevy_ecs/src/schedule/config.rs

```diff
@@ -40,6 +40,7 @@ pub struct SystemConfig {
     pub(super) system: BoxedSystem,
     pub(super) graph_info: GraphInfo,
     pub(super) conditions: Vec<BoxedCondition>,
+    pub(super) ignore_stepping: bool,
```

Comment:
This would probably be clearer as a `SteppingBehavior` enum.

Reply:
What's the rust idiom here?
```rust
enum SteppingBehavior {
   Permit,
   Ignore,
}

// or
enum SteppingBehavior {
    PermitStepping,
    IgnoreStepping,
}
```

The actual code has comments and defaults, but I'm not sure about naming convention here

Reply:
I'm leaning towards the second to permit:
```rust
        use config::SteppingBehavior::*;
        let stepping = match stepping_behavior {
            PermitStepping => true,
            IgnoreStepping => false,
        };
```

Reply:
Also looking for better names here.

I rejected `Enable`/`Disable` because they conflict with enabling/disabling on the `Schedule`.  I want to avoid "oh, you have to enable it both on the system and the schedule".

`Observe`/`Ignore` is an option, but not much better than `Permit`/`Ignore`.

`Observe`/`Disregard`?

`Allow`/`Deny` -- `Deny` feels too much like it could stop stepping from working at all.

Reply:
Maybe

```rust
enum Stepping {
   Incremental,
   Continual,
}
```

Reply:
I just came across `Exempt`, but at the same time, that's a real weird english word, and doesn't have a clear antonym.  Perhaps `Observe`/`Exempt`?